### PR TITLE
feat!: ACIR bytecode encoding with Base64

### PR DESCRIPTION
**Pull request by kobyhallx**
# Description

## Problem\*

Towards https://github.com/noir-lang/acvm/issues/434

## Documentation

- [ ] This PR requires documentation updates when merged.

  

  - [ ] I will submit a noir-lang/docs PR.

  

  - [ ] I will request for and support Dev Rel's help in documenting this PR.

  
  

## Additional Context

# PR Checklist\*

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Comment by kobyhallx** [comment]
Draft, until https://github.com/noir-lang/acvm/pull/436 merges in

**Review by charlielye — DISMISSED** [review]
Looks good.

File: flake.nix [hunk 1]
```diff
@@ -281,6 +281,7 @@
           toml2json
           llvmPackages.lldb # This ensures the right lldb is in the environment for running rust-lldb
           wasm-bindgen-cli
+          jq
```
Comment (TomAFrench) [inline flake.nix:284]: Why is this necessary now?

Comment (kobyhallx) [reply flake.nix:284]: this is not necessary but better to have around in the shell

**Review by charlielye — APPROVED** [review]
